src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -340,9 +340,25 @@ class BugsnagPlugin : Plugin<Project> {
             project.rootProject.allprojects { subProj ->
                 val defaultNodeModulesDir = File("${subProj.rootDir}/../node_modules")
                 val nodeModulesDir = bugsnag.nodeModulesDir.getOrElse(defaultNodeModulesDir)
+                if (!nodeModulesDir.exists()) {
+                    throw StopExecutionException(
+                        "Cannot find node_modules directory at: ${nodeModulesDir.absolutePath}. " +
```

Comment:
```suggestion
                        "Cannot find node_modules directory at: ${nodeModulesDir.absolutePath}" +
```

Wondering if that trailing dot might cause unnecessary confusion?